data/migrations/V0213__add_leadership_pac_linkage.sql

```diff
@@ -0,0 +1,114 @@
+/*
+Issue #4586
+
+Migration file to create cand_cmte_linkage_alternate, leadership_pac_cand_cmte_link 
+*/
+
+-- ------------------------------------------
+-- disclosure.cand_cmte_linkage_alternate
+-- ------------------------------------------
+DO $$
+BEGIN
+	EXECUTE format('CREATE TABLE disclosure.cand_cmte_linkage_alternate
+(
+    sub_id             numeric(19,0) NOT NULL,
+    cand_id            varchar(9) NOT NULL,
+    cand_election_yr   numeric(4,0) NOT NULL,
+    fec_election_yr    numeric(4,0) NOT NULL,
+    cmte_id            varchar(9),
+    cmte_tp            varchar(1),
+    cmte_dsgn          varchar(1),
+    linkage_type       varchar(1) ,
+    cmte_count_cand_yr numeric(2,0),
+    efile_paper_ind    varchar(1),
+    created_by         numeric(12,0)                    DEFAULT  1,
+    created_date       timestamp without time zone      DEFAULT now(),
+    updated_by         numeric(12,0)                    DEFAULT  1,
+    updated_date       timestamp without time zone      DEFAULT now(),
```

Comment:
The default value for "created_by", "created_date", "updated_by" and "updated_date" are in the Oracle side.  
Golden_Gate will transfer the values from the source (Oracle) table to the target (Aurora postgres) table.  With default values available in the source (Oracle), there is no need for setting default in the target.